tests/containers/podman_integration.pm

```diff
@@ -61,6 +60,12 @@ sub run {
     }
     install_packages(@pkgs);
 
+    # Remove CNI packages because an update may install podman without netavark, using CNI as default
+    if (script_run("rpm -q cni") == 0) {
```

Comment:
Careful, CNI is going to be the default only from SLEM 6.0 onward. Even SLES15-SP6 still has podman 4.7.2 and `cni`. SLEM 5.5 has 4.8.3 and `cni` as well.

Reply:
CNI has been deprecated since podman 4.4.0:

https://github.com/containers/podman/blob/main/RELEASE_NOTES.md?plain=1#L719

Reply:
Agreed for 15-SP6+ and SLEM5.5+. For older releases this might cause issues and we have already `podman_integration` tests running there e.g. https://openqa.suse.de/tests/14209375

Reply:
I checked again, we can apply this because we only run those tests on SLEM5.5+ and 15-SP4+.